### PR TITLE
revise expressions pushed down

### PR DESCRIPTION
**Pull request by Yisaer**
Signed-off-by: yisaer <disxiaofei@163.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@Yisaer 是否为 v6.0 相关的呢？请@ tech review 吧，谢谢~

**Comment by xiejiandong** [comment]
lgtm

**Review by winoros — APPROVED** [review]

File: functions-and-operators/expressions-pushed-down.md [hunk 1]
```diff
@@ -16,12 +16,12 @@ aliases: ['/docs-cn/dev/functions-and-operators/expressions-pushed-down/','/docs
...
+| [JSON 运算](/functions-and-operators/json-functions.md) | [JSON_TYPE(json_val)][json_type],<br/> [JSON_EXTRACT(json_doc, path[, path] ...)][json_extract],<br/> [JSON_OBJECT(key, val[, key, val] ...)][json_object],<br/> [JSON_ARRAY([val[, val] ...])][json_array],<br/> [JSON_MERGE(json_doc, json_doc[, json_doc] ...)][json_merge],<br/> [JSON_SET(json_doc, path, val[, path, val] ...)][json_set],<br/> [JSON_INSERT(json_doc, path, val[, path, val] ...)][json_insert],<br/> [JSON_REPLACE(json_doc, path, val[, path, val] ...)][json_replace],<br/> [JSON_REMOVE(json_doc, path[, path] ...)][json_remove] |
 | [日期运算](/functions-and-operators/date-and-time-functions.md) | [`DATE_FORMAT()`](https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_date-format)  |
 
 ## 禁止特定表达式下推
 
-当函数的计算过程由于下推而出现异常时，可通过黑名单功能禁止其下推来快速恢复业务。具体而言，你可以将上述支持的函数或运算符名加入黑名单 `mysql.expr_pushdown_blacklist` 中，以禁止特定表达式下推。
+当上述支持的函数或特定的数据类型（**仅支持** [`ENUM` 类型](/data-type-string.md#enum-类型) 和 [`BIT` 类型](/data-type-numeric.md#bit-类型)）的计算过程由于下推而出现异常时，你可以使用黑名单功能禁止其下推，从而快速恢复 TiDB 业务。具体而言，你可以将函数名，运算符名，或数据列类型加入黑名单 `mysql.expr_pushdown_blacklist` 中，以禁止特定表达式下推。具体方法，请参阅[加入黑名单](#加入黑名单)。
```
Comment (en-jin19) [inline functions-and-operators/expressions-pushed-down.md:24]:
```suggestion
当上述支持的函数或特定的数据类型（**仅限** [`ENUM` 类型](/data-type-string.md#enum-类型) 和 [`BIT` 类型](/data-type-numeric.md#bit-类型)）的计算过程由于下推而出现异常时，你可以使用黑名单功能禁止其下推，从而快速恢复 TiDB 业务。具体而言，你可以将函数名，运算符名，或数据列类型加入黑名单 `mysql.expr_pushdown_blacklist` 中，以禁止特定表达式下推。具体方法，请参阅[加入黑名单](#加入黑名单)。
```

File: functions-and-operators/expressions-pushed-down.md [hunk 2]
```diff
@@ -54,19 +54,23 @@ tidb> desc mysql.expr_pushdown_blacklist;
 
 执行以下步骤，可将一个或多个函数或运算符加入黑名单：
 
-1. 向 `mysql.expr_pushdown_blacklist` 插入对应的函数名或运算符名以及希望禁止下推的存储类型集合。
+1. 向 `mysql.expr_pushdown_blacklist` 插入以下两种类型的内容：
+
+    - 希望禁止下推的[函数名，运算符名](#已支持下推的表达式列表），或数据类型（**仅支持** [`ENUM` 类型](/data-type-string.md#enum-类型) 和 [`BIT` 类型](/data-type-numeric.md#bit-类型)）
```
Comment (en-jin19) [inline functions-and-operators/expressions-pushed-down.md:59]:
```suggestion
    - 希望禁止下推的[函数名，运算符名](#已支持下推的表达式列表），或数据类型（**仅限** [`ENUM` 类型](/data-type-string.md#enum-类型) 和 [`BIT` 类型](/data-type-numeric.md#bit-类型)）
```

File: functions-and-operators/expressions-pushed-down.md [hunk 3]
```diff
@@ -16,12 +16,12 @@ aliases: ['/docs-cn/dev/functions-and-operators/expressions-pushed-down/','/docs
...
+| [JSON 运算](/functions-and-operators/json-functions.md) | [JSON_TYPE(json_val)][json_type],<br/> [JSON_EXTRACT(json_doc, path[, path] ...)][json_extract],<br/> [JSON_OBJECT(key, val[, key, val] ...)][json_object],<br/> [JSON_ARRAY([val[, val] ...])][json_array],<br/> [JSON_MERGE(json_doc, json_doc[, json_doc] ...)][json_merge],<br/> [JSON_SET(json_doc, path, val[, path, val] ...)][json_set],<br/> [JSON_INSERT(json_doc, path, val[, path, val] ...)][json_insert],<br/> [JSON_REPLACE(json_doc, path, val[, path, val] ...)][json_replace],<br/> [JSON_REMOVE(json_doc, path[, path] ...)][json_remove] |
 | [日期运算](/functions-and-operators/date-and-time-functions.md) | [`DATE_FORMAT()`](https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_date-format)  |
 
 ## 禁止特定表达式下推
 
-当函数的计算过程由于下推而出现异常时，可通过黑名单功能禁止其下推来快速恢复业务。具体而言，你可以将上述支持的函数或运算符名加入黑名单 `mysql.expr_pushdown_blacklist` 中，以禁止特定表达式下推。
+当上述的函数，运算符，或特定的数据类型（**仅限** [`ENUM` 类型](/data-type-string.md#enum-类型) 和 [`BIT` 类型](/data-type-numeric.md#bit-类型)）的计算过程因下推而出现异常时，你可以使用黑名单功能禁止其下推，从而快速恢复 TiDB 业务。具体而言，你可以将函数名，运算符名，或数据列类型加入黑名单 `mysql.expr_pushdown_blacklist` 中，以禁止特定表达式下推。具体方法，请参阅[加入黑名单](#加入黑名单)。
```
Comment (TomShawn) [inline functions-and-operators/expressions-pushed-down.md:24]:
```suggestion
当[已支持下推的表达式列表](#已支持下推的表达式列表)中的函数和运算符，或特定的数据类型（**仅限** [`ENUM` 类型](/data-type-string.md#enum-类型) 和 [`BIT` 类型](/data-type-numeric.md#bit-类型)）的计算过程因下推而出现异常时，你可以使用黑名单功能禁止其下推，从而快速恢复 TiDB 业务。具体而言，你可以将函数名、运算符名，或数据列类型加入黑名单 `mysql.expr_pushdown_blacklist` 中，以禁止特定表达式下推。具体方法，请参阅[加入黑名单](#加入黑名单)。
```

File: functions-and-operators/expressions-pushed-down.md [hunk 4]
```diff
@@ -39,8 +39,8 @@ tidb> desc mysql.expr_pushdown_blacklist;
 
 以上结果字段解释如下：
 
-+ `name`：禁止下推的函数名。
-+ `store_type`：用于指明希望禁止该函数下推到哪些组件进行计算。组件可选 `tidb`、`tikv` 和 `tiflash`。`store_type` 不区分大小写，如果需要禁止向多个存储引擎下推，各个存储之间需用逗号隔开。
++ `name`：禁止下推的函数名，运算符名，或数据类型。
```
Comment (TomShawn) [inline functions-and-operators/expressions-pushed-down.md:42]:
```suggestion
+ `name`：禁止下推的函数名、运算符名，或数据类型。
```

File: functions-and-operators/expressions-pushed-down.md [hunk 5]
```diff
@@ -39,8 +39,8 @@ tidb> desc mysql.expr_pushdown_blacklist;
...
++ `store_type`：用于指明希望禁止该函数，运算符，或数据类型下推到哪些组件进行计算。组件可选 `tidb`、`tikv` 和 `tiflash`。`store_type` 不区分大小写，如果需要禁止向多个存储引擎下推，各个存储之间需用逗号隔开。
```
Comment (TomShawn) [inline functions-and-operators/expressions-pushed-down.md:43]:
```suggestion
+ `store_type`：用于指定希望禁止该函数、运算符、数据类型下推到哪些组件进行计算。组件可选 `tidb`、`tikv` 和 `tiflash`。`store_type` 不区分大小写，如果需要禁止向多个存储引擎下推，各个存储之间需用逗号隔开。
```

File: functions-and-operators/expressions-pushed-down.md [hunk 6]
```diff
@@ -54,19 +54,23 @@ tidb> desc mysql.expr_pushdown_blacklist;
 
 执行以下步骤，可将一个或多个函数或运算符加入黑名单：
 
-1. 向 `mysql.expr_pushdown_blacklist` 插入对应的函数名或运算符名以及希望禁止下推的存储类型集合。
+1. 向 `mysql.expr_pushdown_blacklist` 插入以下内容：
+
+    - 希望禁止下推的[函数名，运算符名](#已支持下推的表达式列表），或数据类型（**仅限** [`ENUM` 类型](/data-type-string.md#enum-类型) 和 [`BIT` 类型](/data-type-numeric.md#bit-类型)）
```
Comment (TomShawn) [inline functions-and-operators/expressions-pushed-down.md:59]:
```suggestion
    - 希望禁止下推的[函数名、运算符名](#已支持下推的表达式列表)或数据类型（**仅限** [`ENUM` 类型](/data-type-string.md#enum-类型) 和 [`BIT` 类型](/data-type-numeric.md#bit-类型)）
```

File: functions-and-operators/expressions-pushed-down.md [hunk 7]
```diff
@@ -54,19 +54,23 @@ tidb> desc mysql.expr_pushdown_blacklist;
...
+    - 希望禁止下推的存储引擎
+
 2. 执行 `admin reload expr_pushdown_blacklist;`。
 
 ### 移出黑名单
 
 执行以下步骤，可将一个或多个函数及运算符移出黑名单：
 
-1. 从 `mysql.expr_pushdown_blacklist` 表中删除对应的函数名或运算符名。
+1. 从 `mysql.expr_pushdown_blacklist` 表中删除对应的函数名，运算符名，或数据类型。
```
Comment (TomShawn) [inline functions-and-operators/expressions-pushed-down.md:68]:
```suggestion
1. 从 `mysql.expr_pushdown_blacklist` 表中删除对应的函数名、运算符名或数据类型。
```

File: functions-and-operators/expressions-pushed-down.md [hunk 8]
```diff
@@ -54,19 +54,23 @@ tidb> desc mysql.expr_pushdown_blacklist;
...
 2. 执行 `admin reload expr_pushdown_blacklist;`。
 
-### 黑名单用法示例
+### 黑名单功能使用示例
```
Comment (TomShawn) [inline functions-and-operators/expressions-pushed-down.md:71]:
```suggestion
### 黑名单使用示例
```

File: system-variables.md [hunk 9]
```diff
@@ -1652,3 +1652,9 @@ set tidb_slow_log_threshold = 200;
 - 作用域：SESSION | GLOBAL
 - 默认值：`ON`
 - 这个变量用于控制计算窗口函数时是否采用高精度模式。
+
+### `tidb_sysdate_is_now`
+
+- 作用域：SESSION | GLOBAL
+- 默认值：`OFF`
+- 这个变量用于控制 `sydate` 函数是否替换为 `NOW` 函数，效果与 MYSQL 中的 [`sysdate-is-now`](https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_sysdate-is-now) 一致。
```
Comment (en-jin19) [inline system-variables.md:1660]:
```suggestion
- 这个变量用于控制 `SYSDATE` 函数是否可以替换为 `NOW` 函数，效果与 MYSQL 中的 [`sysdate-is-now`](https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_sysdate-is-now) 一致。
```

File: system-variables.md [hunk 10]
```diff
@@ -1652,3 +1652,9 @@ set tidb_slow_log_threshold = 200;
 - 作用域：SESSION | GLOBAL
 - 默认值：`ON`
 - 这个变量用于控制计算窗口函数时是否采用高精度模式。
+
+### `tidb_sysdate_is_now`
```
Comment (en-jin19) [inline system-variables.md:1656]:
```suggestion
### `tidb_sysdate_is_now` <span class="version-mark">从 v6.0.0 版本开始引入</span>
```

File: system-variables.md [hunk 11]
```diff
@@ -1652,3 +1652,9 @@ set tidb_slow_log_threshold = 200;
 - 作用域：SESSION | GLOBAL
 - 默认值：`ON`
 - 这个变量用于控制计算窗口函数时是否采用高精度模式。
+
+### `tidb_sysdate_is_now` <span class="version-mark">从 v6.0.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 默认值：`OFF`
+- 这个变量用于控制 `SYSDATE` 函数是否可以替换为 `NOW` 函数，效果与 MYSQL 中的 [`sysdate-is-now`](https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_sysdate-is-now) 一致。
```
Comment (en-jin19) [inline system-variables.md:1660]:
```suggestion
- 这个变量用于控制 `SYSDATE` 函数能否替换为 `NOW` 函数，其效果与 MYSQL 中的 [`sysdate-is-now`](https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_sysdate-is-now) 一致。
```

File: system-variables.md [hunk 12]
```diff
@@ -1652,3 +1652,9 @@ set tidb_slow_log_threshold = 200;
 - 作用域：SESSION | GLOBAL
 - 默认值：`ON`
 - 这个变量用于控制计算窗口函数时是否采用高精度模式。
+
+### `tidb_sysdate_is_now` <span class="version-mark">从 v6.0.0 版本开始引入</span>
```
Comment (TomShawn) [inline system-variables.md:1656]:
```suggestion
### `tidb_sysdate_is_now`（从 v6.0.0 版本开始引入）
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
tidb_rc_read_check_ts 较多说明以及使用限制放在变量文档里，有点塞不下的感觉，而且也不易查找。能否考虑将特性说明放在 https://docs.pingcap.com/zh/tidb/dev/transaction-isolation-levels 这里或是哪篇文档里，而变量文档只需简单介绍变量即可?

**Pull request merged**